chapters/annotations.tex

```diff
@@ -1747,13 +1747,15 @@ \subsubsection{Conversion Rules}\label{conversion-rules}
 The conversion should generate correct Modelica models using the new version of the library
 corresponding to the old version.
 
+The string arguments for these functions should not be the empty string (except for the string in an empty array of strings).
```

Comment:
Two things:
- The decision was to forbid, so _shall_.
- I don't understand how an empty array of strings may contain an empty string.  if the comment has to do with `fill("", 0)`, I don't see the point of trying to clarify.

Reply:
I have removed this entire part as it was duplicated, and used the previously suggested text.

Reply:
@henrikt-ma ?